### PR TITLE
works with symfony3

### PR DESCRIPTION
**Pull request by gam6itko**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "hanfrey/sparkpost-bundle",
+  "name": "gam6itko/sparkpost-bundle",
```
Comment (sadortun) [inline composer.json]: You should keep the original package name for the PR

**Comment by gam6itko** [comment]
bump

**Comment by florentdestremau** [comment]
oh actually yes, bump @sadortun

**Comment by sadortun** [comment]
@florentdestremau  ?

**Comment by florentdestremau** [comment]
sorry, saw you as reviewer ^^, I guess it would make more sense pinging @Hanfrey

**Comment by Hanfrey** [comment]
Merged

**Comment by florentdestremau** [comment]
Was that automatic ?

**Comment by florentdestremau** [comment]
Or just a coincidence :laughing:

**Comment by Hanfrey** [comment]
Coincidence :)

**Pull request merged**